akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -95,6 +118,7 @@ sealed trait HttpEntity extends jm.HttpEntity {
 /* An entity that can be used for body parts */
 sealed trait BodyPartEntity extends HttpEntity with jm.BodyPartEntity {
   def withContentType(contentType: ContentType): BodyPartEntity
+  def withSizeLimit(maxBytes: Long): BodyPartEntity
```

Comment:
That's a user facing API method as well right? More rarely used perhaps but still.
Copying the comment from is a bit of a pain, so let's at least add a `See [[HttpEntity#withSizeLimit]] for detailed documentation.`?